src/telemetry/recvloop.cpp

```diff
@@ -38,63 +38,41 @@ RecvLoop::RecvLoop(Logger &log, data::Data& data, Main* main_pointer)
 void RecvLoop::run()
 {
   log_.DBG("Telemetry", "Telemetry RecvLoop thread started");
-  telemetry_data::ServerToClient msg;
-  // not sure whether to put this in or ouside of loop
+
   data::Telemetry telem_data_struct = data_.getTelemetryData();
 
   while (true) {
-    try {
-      msg = main_ref_.client_.receiveData();
-    }
-    catch (std::exception& e) {
-      log_.ERR("Telemetry", "%s", e.what());
+    std::string message = main_ref_.client_.receiveData();
```

Comment:
`receiveData()` can still throw an exception, probably best to put the try catch block back in here

Reply:
Good point, I'll add that.